client/java-spring-boot-autoconfigure/src/test/java/com/linecorp/centraldogma/client/spring/CentralDogmaInitializationTimeoutTest.java

```diff
@@ -56,16 +61,31 @@ static class TestConfiguration {}
     @Inject
     private CentralDogma client;
 
+    @Timeout(value = 5, unit = TimeUnit.MINUTES)
     @BeforeAll
     static void beforeAll() {
         lock.lock();
-        server = Server.builder()
-                       .http(TEST_SERVER_PORT)
-                       .service(HttpApiV1Constants.HEALTH_CHECK_PATH,
-                                (ctx, req) -> HttpResponse.delayed(HttpResponse.of("OK"),
-                                                                   Duration.ofSeconds(5)))
-                       .build();
-        server.start().join();
+        final int maxAttempts = 8;
+        for (int i = 1; i <= maxAttempts; i++) {
+            try {
+                final Server server =
+                        Server.builder()
+                              .http(TEST_SERVER_PORT)
```

Comment:
Question) Is it possible to randomize the port number as well?

Reply:
I have no idea how to randomize it at the moment. The port number is set in the following configuration.
https://github.com/line/centraldogma/blob/07bae6588f88600efa7d9de376e3ad854f16fc8d/client/java-spring-boot-autoconfigure/src/test/resources/config/application-initTimeout.yml#L3
I need to research if we can dynamically update the value.

Reply:
Ah forgot about this. Fair enough 😅